pkgs/applications/science/biology/aragorn/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchurl }:
```

Comment:
```suggestion
{ stdenv, fetchurl }:
```

Reply:
OK, I commited the suggestion. Should I squash the 2 commits and force-push ?

Reply:
Please.